vignettes/git_usage.Rmd

```diff
@@ -45,8 +45,6 @@ Each feature branch must be related to an issue. We encourage new developers to
 
 The name of the branch must be prefixed with the issue number, followed by a short but meaningful description and the `@<target_branch>` suffix. The latter would be `@devel` in most cases. As an example, given an issue #94 "Program function to derive `LSTALVDT`", the branch name would be `94_derive_var_lstalvdt@devel`.
 
-The `@<target_branch>` suffix is used in our CI/CD pipelines, e.g. when running `R CMD check`. It ensures that `{admiral}`'s dependencies such as `{admiral.test}` and `{admiraldev}` are installed from the specified target branch. So when the target branch is set to `@devel` the dependencies will be installed from those package's respective `devel` branches rather than installing the latest released version. This ensures that we test the development version of `{admiral}` against the development versions of its  dependencies. That way all packages are kept in sync.
```

Comment:
Why is this paragraph removed?

I would keep it and just replace "admiral.test" with "pharmasdtm".

Reply:
oh my mistake, it was a bad copy & paste job